documentation/governance.md

```diff
@@ -59,22 +57,26 @@ Distinct branches of the Standard will be maintained within Github for each vers
 * Development – indicated by a -dev suffix (e.g. 1.0-dev). Both schema and documentation on a development branch can be updated and should only be implemented on an experimental basis.
 * Live – with no suffix (e.g. 1.0). Only documentation updates are permitted on a live branch. All documentation changes must be reviewed to ensure they do not make any changes to the meaning of the Standard.
 
-Semantic Versioning practices will be used to distinguish between:
-* Major versions which make backwards-incompatible API changes; and
-* Minor versions which add functionality in a backwards-compatible manner.
+[Semantic Versioning](https://semver.org/) practices will be used to distinguish between:
+* MAJOR versions which make backwards-incompatible API changes; and
+* MINOR versions which add functionality in a backwards-compatible manner.
+* PATCH version which make backwards-compatible bug fixes.
 
-These are captured by a version number in the format MAJOR.MINOR
+These are captured by a version number in the format MAJOR.MINOR.PATCH.
 
 ### Revision process
-To release a new minor or major version upgrade will involve a number of stages outlined in the flowchart below, and described in more depth in the following sections.
+To release a new MINOR or MAJOR version upgrade will involve a number of stages outlined in the flowchart below, and described in more depth in the following sections.
 
 ![Revision process](../assets/upgrade_process_feb_2016.png)
 
+PATCH version upgrades are a smaller process, in recognition of the low impact that PATCH changes can have on the whole standard. A proposal for a PATCH version upgrade is made via the forum, with the Stewardship Comittee and technical team notified via email. If no objections are received within one week, the PATCH change will be considered approved.
```

Comment:
In the google doc - https://docs.google.com/document/d/15a5V5-SB5X2GLu--2XizXulowX7yxzeZMS4vTnxJWe8/edit This paragraph comes after the list below, not before.

Reply:
After I read it again, I though this was a better place for it. I'm happy to match the Google doc precisely if that's what we're going for, but I think this reflects what was said at the Committee meeting more closely, and makes the document flow better.